jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css

```diff
@@ -0,0 +1,163 @@
+.splitScreen {
+    height: calc(100vh);
+    display: flex;
+    flex-direction: row;   
+}
+
+.leftSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+    background-color: #023C47;
+}
+
+.loginRectangle {
+    width: 505px;
+    height: 701px;
+    top: 216px;
+    left: 125px;
+    border-radius: 10px;
+    border: 0.5px;
+    background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
+    border: 0.5px solid #878787
+}
+
+.rightSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+}
+
+.rightImage {
+    width: 50%;
+    height: 50%;
+}
+
+.alignWelcome {
+    text-align: left;
+    padding-top: 80px;
+    padding-left: 43px;
+    font-family: 'DM Serif Text', serif;
+    font-size: 31px;
+}
+
+.alignWelcomeBlurb {
+    text-align: left;
+    font-family: 'Inter', serif;
+    padding-left: 43px;
+    padding-right: 43px;
+    font-size: 16px;
+
+}
+
+.alignWelcome, .alignWelcomeBlurb {
+    margin-bottom: -25px; /* Adjust this value as needed to control the spacing */
```

Comment:
Using negative margin is generally considered bad practice as it makes the code more complex and difficult to understand. You can definitely style page the same way by only using positive values for margin/padding. It would be helpful if you could update that

Reply:
Additionally, consider using units like percentages or  vh/vw or rem for spacing / sizing properties instead of px since this is supposed to help make the page more responsive to different sizes